plugin/testdata/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM golang:latest AS builder
+
+WORKDIR /fluent-bit-go
+
+COPY go.mod .
+COPY go.sum .
+
+RUN go mod download
+RUN go mod verify
+
+COPY . .
+
+RUN go build -trimpath -buildmode c-shared -o ./bin/go-test-input-plugin.so ./plugin/testdata/input/input.go
+RUN go build -trimpath -buildmode c-shared -o ./bin/go-test-output-plugin.so ./plugin/testdata/output/output.go
+
+FROM ghcr.io/calyptia/enterprise/advanced:main
```

Comment:
I think this got renamed to calyptia/calyptia-fluent-bit/advanced:main

Reply:
The repo was renamed but I see that the images have the old name still 🤔

Reply:
Nevermind. It did pull :)

Reply:
Done